### DISCUSSION TITLE
How to change Flet icon or set to a custom icon in windows desktop app's title bar

### DISCUSSION THREAD

**Discussion by TheCodeCache** [flet-dev/flet#1030] [Q&A] [answered]
Here in this below sample `Flet desktop app`, there is this flet `icon` <img src="https://user-images.githubusercontent.com/26399543/218325747-dda52e23-be74-43ad-b66b-7b09230296ca.png" width="20" height="18"> displayed at the top left corner in `windows title bar`.

<img width="362" alt="image" src="https://user-images.githubusercontent.com/26399543/218325582-3d750655-7c10-48c1-b5ee-d1a83cca55ab.png">

I've below query as follows - 

1. How could we remove that or set to null ?
2. How to set to a different/custom icon / image?

Thanks in advance!

**Comment by TheCodeCache** [comment]
Please suggest!

**Comment by iron3oxide** [comment, marked as answer]
Check out #991, it should contain all the information you need. If you still have questions, just reply.

**Reply by TheCodeCache** [reply]
Explored #991 and got the idea, I'm good now : )
Thanks,

**Reply by iron3oxide** [reply]
No problem. Could you mark this thread as the answer to your question? It's always nice to be able to quickly tell that a question has been answered and what the answer is.

**Reply by TheCodeCache** [reply]
Sure @iron3oxide !,

The below comment on #991 answered my queries - 

<img width="623" alt="image" src="https://user-images.githubusercontent.com/26399543/218359902-737a377f-e431-4ada-a7cf-524589b270aa.png">

Thanks for the help

**Comment by kacper-the-programmer** [comment]
https://flet.dev/docs/reference/types/window#icon

**Reply by ndonkoHenri** [reply]
I added this feature but haven't tested it yet because it is windows-only.
Have you tried it?

**Reply by kacper-the-programmer** [reply]
yep, and it works

**Reply by ndonkoHenri** [reply]
@Kacper-OS will you mind helping https://github.com/flet-dev/flet/issues/3438 ?

**Reply by kacper-the-programmer** [reply]
sure
